code/game/objects/items/weapons/storage/backpack.dm

```diff
@@ -97,6 +97,71 @@
 	name = "trophy rack"
 	desc = "It's useful for both carrying extra gear and proudly declaring your insanity."
 	icon_state = "cultpack"
+	var/obj/item/organ/external/head/LS
+	var/obj/item/organ/external/head/MS
+	var/obj/item/organ/external/head/RS
+	var/mutable_appearance/LSma
+	var/mutable_appearance/MSma
+	var/mutable_appearance/RSma
+	var/list/placement
+
+/obj/item/storage/backpack/cultpack/attackby(obj/item/I, mob/living/user)
+	..()
+	if(istype(I, /obj/item/organ/external/head))
+		placement = input(user, "Where would you like to put the head?") in list(!LS ? "Left Spike" : "", !MS ? "Middle Spike" : "", !RS ? "Right Spike" : "", "Bag")
+		switch(placement)
```

Comment:
Need to check after if the spot is still viable
![image](https://user-images.githubusercontent.com/15887760/84575547-1109ef80-adae-11ea-920e-4e798d0e97ba.png)

Reply:
Fixed